global/cta-analytics@1.0.js

```diff
@@ -0,0 +1,41 @@
+!function () {
+  window.$loaded(function (window, document, $, undefined) {
+    var ready = function () {
+      (function trackCTAAnalytics() {
+        function trackCTAElement() {
+          var link = $(this);
+          var name = link.attr('data-name');
+          var category = link.attr('data-category');
+          var action = link.attr('data-action');
+          var source = link.attr('data-source');
+          var label = link.attr('data-label');
+          var source_detail = link.attr('data-source-detail');
+
+          var properties = { category, action, source, label, source_detail };
+
+          link.on('click', function (e) {
+            //disable clicks
+            link.attr('style', 'pointer-events: none');
```

Comment:
It's better for accessibility to use the `disabled` attribute
```suggestion
            link.attr('disabled', true);
```